Automattic/vip-go-mu-plugins
# don't validate count without index

## Description

This adds another condition to skipping count validation. Before this change count validations would through an alert if the index doesnt exist.

## Changelog Description


### Plugin Updated: Search

Improve internal index count monitoring job to skip non-initialized indexes.


## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Have a multisite environment
2. add subsite (this creates site with index-v1 only in DB, but not in ES)
3. Run `wp vip-search health validate-counts --network-wide`
4. You should get a message about index skipped due to index not being in ES.
